public/scripts/WebSocket.jsx

```diff
@@ -56,6 +57,16 @@ function getAuthToken(auth_url) {
 }
 
 
+function clearNotifications(dispatch, tag) {
+  return new Promise(
+    (resolve, reject) => {
+      dispatch(hideNotificationByTag(tag));
+      resolve();
+    }
+  );
+}
+
```

Comment:
@stefanv I was considering moving this function to Notifications.jsx what do you think?